htdocs/holiday/class/holiday.class.php

```diff
@@ -1692,27 +1691,86 @@ public function updateSoldeCP($userID = 0, $nbHoliday = 0, $fk_type = 0)
 					$nowHoliday = $userCounter['nb_holiday'];
 					$newSolde = $nowHoliday + $nbDaysToAdd;
 
-					// We add a log for each user
-					$this->addLogCP($user->id, $userCounter['rowid'], $langs->trans('HolidaysMonthlyUpdate'), $newSolde, $userCounter['type']);
+					// We add a log for each user when its balance gets increased
+					$this->addLogCP($user->id, $userCounter['rowid'], $langs->trans('HolidayMonthlyCredit'), $newSolde, $userCounter['type']);
```

Comment:
What is the goal of using the translation key HolidayMonthlyCredit compared to HolidayMonthlyUpdate ?

Reply:
The goal is to be more understandable when reading the logs of the update because update could mean increasing or decreasing. Now, the user would know if his balance is credited or debited.